src/earn/EarnHome.test.tsx

```diff
@@ -196,7 +196,7 @@ describe('EarnHome', () => {
     ).toBeTruthy()
 
     fireEvent.press(getByText('tokenBottomSheet.filters.tokens'))
-    fireEvent.press(getByTestId('USDCSymbol'))
+    fireEvent.press(getByTestId('BottomSheetUSDCSymbol'))
```

Comment:
This was specifically for e2e tests, which I struggled with for a while. Since we moved the quick actions to the wallet screen, there would be two elements with the `cUSDSymbol` test ID. Strangely, when both the bottom sheet and the wallet balances were showing, the tests were unable to find even a single element with the given test ID. Using the helper fn we have to grab elements at a particular index (for presumably this very use case) also failed to find any elements..